packages/dashboards/addon/components/navi-widget.js

```diff
@@ -45,16 +45,22 @@ export default Component.extend({
   /**
    * @property {Object} options - object for grid-stack-item
    */
-  options: computed('layoutOptions', function() {
-    let layout = get(this, 'layoutOptions'),
-      id = get(this, 'model.id');
+  options: computed(
+    'layoutOptions.@each.{width,height,row,column,widgetId}',
+    'layoutOptions',
+    'layoutOptions.[]',
```

Comment:
Can we remove lines 50 & 51? 49 takes care of it?

https://guides.emberjs.com/v3.12.0/object-model/computed-properties-and-aggregate-data/#toc_each